backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc

```diff
@@ -60,134 +75,11 @@ one of the must-execute tests for ensuring backward compatibility between EAP8+
 == Community Documentation
 
 Community documentation will be provided for how the aforementioned compatibility among remote EE clients and servers.
-It will be based on this analysis document as reference.
+It will be added to the WildFly documentation, and it will be based on this analysis document as reference.
+
+== Release Note Content
+Bullet point: support interoperability between Jakarta EE 9 servers and Java EE 8 or older servers
 
 == Design Notes
 
-The backward compatibility between two Java EE 8- and Jakarta EE 9+ remote endpoints has to be enabled at the endpoint
```

Comment:
@bstansberry  Is it correct to move the design notes from here to a Google document? I followed the RFE process document that mentions there is an optional design document deliverable., since there is no Design Notes in the template used by the RFE process document.

Reply:
I'll wait for @bstansberry response to this question before approving and merging @fl4via